astropy/modeling/tests/test_functional_models.py

```diff
@@ -153,6 +153,16 @@ def test_Shift_inverse():
     assert_allclose(m.inverse(m(6.789)), 6.789)
 
 
+def test_Scale_fittable():
+    m = models.Scale(1.2)
+    assert(m.fittable==True)
+
+
+def test_Shift_fittable():
+    m = models.Shift(1.2)
+    assert(m.fittable==True)
+
+
```

Comment:
Yah, I would not spend too much time on this. The point of making them `fittable` is to be able to include them in compound models which are `fittable`.